### ISSUE TITLE
Ansible2: "Ambiguous output redirect." error during TASK [setup]

### ISSUE THREAD

**Issue by yeroc** [ansible/ansible#12551]
##### Issue Type:
- Bug Report
##### Ansible Version:

```
ansible 2.0.0 (devel 7a4266e9c5) last updated 2015/09/28 13:51:05 (GMT -600)
  lib/ansible/modules/core: (detached HEAD a4ac171a07) last updated 2015/09/28 13:51:12 (GMT -600)
  lib/ansible/modules/extras: (detached HEAD 3c8c716801) last updated 2015/09/28 13:51:14 (GMT -600)
  config file = /etc/ansible/ansible.cfg
  configured module search path = None
```
##### Ansible Configuration:

None.
##### Environment:

Management Host: CentOS release 6.5 x64
Target Host: CentOS release 6.7 x64
##### Summary:

After upgrading to Ansible 2 a simple play fails during the setup stage with an "Ambiguous output redirect." error.  This simple play works fine with 1.9.x.
##### Steps To Reproduce:

Given a simple play:

``` yaml

---
- hosts: webserver
  remote_user: "{{ deploy_user }}"
  sudo: no
  tasks:
  - name: debug
    debug: msg="Hello world!"
```
##### Expected Results:

This is what we get with Ansible 1.9:

```
$ ansible-playbook -i inventory connect-failure.yml 

PLAY [webserver] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [inet-dmz2-tst]

TASK: [debug] ***************************************************************** 
ok: [inet-dmz2-tst] => {
    "msg": "Hello world!"
}

PLAY RECAP ******************************************************************** 
inet-dmz2-tst              : ok=2    changed=0    unreachable=0    failed=0   
```
##### Actual Results:

But with Ansible 2:

```
$ ansible-playbook -i inventory connect-failure.yml 

PLAY ***************************************************************************

TASK [setup] *******************************************************************
fatal: [inet-dmz2-tst]: FAILED! => {"changed": false, "failed": true, "msg": "Ambiguous output redirect.\r\n", "parsed": false}

PLAY RECAP *********************************************************************
inet-dmz2-tst              : ok=0    changed=0    unreachable=0    failed=1   
```

**Comment by yeroc** [comment]
There's a posting to the [google group](https://groups.google.com/forum/#!msg/ansible-project/-cuOaxNnbD4/8sWHAjyeLAAJ) with the same error message and @bcoca mentioned a patch that wasn't applied to current dev.  The target user in my case is indeed using the `csh` shell instead of `bash` so it could well be that it's the same issue.  If so, I'm not sure why the patch mentioned hasn't been applied.

Thanks!

**Comment by yeroc** [comment]
Will this regression be fixed?

**Comment by jimi-c** [comment]
Hi, this issue appears to be resolved, as the commit mentioned above was merged in and has been available in 2.0 since its release.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular issue is resolved.

Thank you!
